### ISSUE TITLE
It is not possible to create two commands in parallel if they are assigned to one connection

### ISSUE THREAD

**Issue by BoBiene** [mysql-net/MySqlConnector#867]
````c#
 MySqlConnection connection = new MySqlConnection(/* ... */);
connection.Open(); ;

var cmd1 = connection.CreateCommand();
cmd1.CommandText = "SELECT 1";
using (var reader = cmd1.ExecuteReader())
{
    var cmd2 = connection.CreateCommand();
    cmd2.CommandText = "SELECT 1";  //should not fail

    //cmd2.Execute[..]() <- should fail
}
````

**Comment by bgrainger** [comment]
The test here is too strict; it really should only be checking whether there is an active reader for _this_ `MySqlCommand`, not any command. https://github.com/mysql-net/MySqlConnector/blob/058c7f51a261d42b320d264c8495f28dcff5f69a/src/MySqlConnector/MySqlCommand.cs#L182-L183

**Comment by bgrainger** [comment]
I suspect this should be fixed by checking `ServerSession.m_activeCommandId`.

**Comment by bgrainger** [comment]
The fix is available as a pre-release, 1.0.1-beta.0.20, from GitHub Packages: https://github.com/mysql-net/MySqlConnector/packages/39735

**Comment by BoBiene** [comment]
Thank you @bgrainger 👍

**Comment by bgrainger** [comment]
Fixed in [1.0.1](https://github.com/mysql-net/MySqlConnector/releases/tag/1.0.1).
